base/setup/usetup/consup.c

```diff
@@ -321,6 +321,47 @@ CONSOLE_SetTextXY(
         &Written);
 }
 
+/**
+ * @CONSOLE_SetTextByID
+ *
+ * Prints a text to the output - screen.
+ *
+ * @param[in]   page
+ *     The MUI (Multilingual User Interface) entry page number, as a unsigned long integer.
+ *
+ * @param[in]   TextID
+ *      The text identification number (ID), as a unsigned integer.
+ *
+ * @return
+ *     Nothing.
+ *
+ * @remarks
+ *      Unlike CONSOLE_SetTextXY, CONSOLE_SetTextByID outputs a portion of text given its text ID and
+ *      entry page interface. The routine calls MUIGetEntryData() to automatically query the entry page data
```

Comment:
This second sentence doesn't seem necessary for the usage of this function.
If somebody is interested this can be observed by just reading the source of the function.